src/Sylius/Bundle/ApiBundle/Swagger/AcceptLanguageHeaderDocumentationNormalizer.php

```diff
@@ -36,8 +40,12 @@ public function normalize($object, $format = null, array $context = [])
             'in' => 'header',
             'required' => false,
             'schema' => [
-                'type' => 'string'
-            ]
+                'type' => 'string',
+                'enum' => array_map(
+                    fn (LocaleInterface $locale): string => $locale->getCode(),
+                    $this->localeRepository->findAll()
```

Comment:
is it a bit misleading? because users can put whatever locale want but it works only for locales enabled in the channel. what do you think about getting a channel and showing available options? or as a minimum improvement, you can add information that these locales are ALL locales declared in Shop

Reply:
the comment with the information that these are all the locales available, but only enabled ones would work in the given channel in the shop could be a nice improvement.

Still it's much better than before. Thanks Ernest!

Reply:
Ok guys I'll improve this a little 😃

Reply:
@lchrusciel, @AdamKasp done 🚀